e2e_playwright/st_audio_input_test.py

```diff
@@ -136,6 +137,27 @@ def test_audio_input_label_visibility_snapshot(
     )
 
 
+@pytest.mark.only_browser("chromium")
+def test_audio_input_file_download(app: Page):
```

Comment:
Related to @lukasmasuch's comment about issues in iframes, we have a download test for dataframes that also tests this in an iframed app to ensure that it works: https://github.com/streamlit/streamlit/blob/develop/e2e_playwright/st_dataframe_interactions_test.py#L253-L323
I think a similar test should be added here. Maybe there is also some opportunity to move it to our shared app_utils and share the logic 🙂